x-pack/plugins/monitoring/public/components/chart/monitoring_timeseries_container.tsx

```diff
@@ -24,11 +19,38 @@ import {
   EuiTextAlign,
   EuiButtonEmpty,
 } from '@elastic/eui';
-import { FormattedMessage } from '@kbn/i18n-react';
-import { i18n } from '@kbn/i18n';
+import { getTechnicalPreview } from './get_technical_preview';
+import { getTitle } from './get_title';
+import { getUnits } from './get_units';
+import { MonitoringTimeseries } from './monitoring_timeseries';
+import { InfoTooltip } from './info_tooltip';
 import { AlertsBadge } from '../../alerts/badge';
+import type { AlertsByName } from '../../alerts/types';
+
+import './monitoring_timeseries_container.scss';
+
+interface ZoomInfo {
+  showZoomOutBtn: () => boolean;
+  zoomOutHandler: () => void;
+}
+
+interface SeriesAlert {
+  alerts: AlertsByName;
+}
+interface Series {
+  metric: { title: string; label: string; description: string };
+}
+interface Props {
+  series?: Series[] | SeriesAlert;
```

Comment:
Everywhere I looked, `series` would receive an array containing metrics. I haven't found where an alert could be passed, but since [here](https://github.com/elastic/kibana/pull/145719/files#diff-bf8dda4e6b7f460b3711973843532969861fbd9698bb6eb98cb0dfd506291e71L93) it verifies whether the `alerts` prop isn't null, I decided to keep as it is with this weird union type